store.config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  // Ecommerce Platform
+  platform: 'vtex',
+
+  // Platform specific configs for API
+  api: {
+    storeId: 'fashioneurope',
+    environment: 'vtexcommercestable',
+  },
+
+  // Default channel
+  channel: '1',
+
+  // Production URLs
+  storeUrl: 'https://www.vtex-base1.tk',
+  checkoutUrl: 'https://chk.vtex-base1.tk/checkout',
+
+  // Lighthouse CI
+  lighthouse: {
+    server: process.env.BASE_SITE_URL || 'http://localhost:9000',
+    urls: ['/', '/women', '/organza-sleeve-top-138/p'],
```

Comment:
To keep it consistent with the cypress config

```suggestion
    pages: ['/', '/women', '/organza-sleeve-top-138/p'],
```